ui/jstests/learningresources/test_learning_resource_panel.jsx

```diff
@@ -482,6 +487,7 @@ define(["QUnit", "react", "test_utils", "jquery", "lodash",
           <LearningResourcePanel
             repoSlug="repo"
             closeLearningResourcePanel={function() {}}
+            markDirty={function() {}}
```

Comment:
Can you add another test which tests `markDirty`? The test can just make changes to the term state, and the description, and assert that the dirty flag was set properly.